### DISCUSSION TITLE
HIKING DTS238-7 WIFI (3 phase) Support

### DISCUSSION THREAD

**Discussion by g6094199** [arendst/Tasmota#16913] [General]
Hi.

Since the device is based on [TYWE3S](https://tasmota.github.io/docs/devices/TYWE3S/) i have already successfully flashed the unit with tasmota.  
![image_2022-10-25_18-46-06](https://user-images.githubusercontent.com/37833326/197856992-27b8f1ee-1dfd-46fe-8424-6dea424db236.jpg)

My ttl converter wasnt able to provide enough power to flash while it was installed, i had to desolder the ESP module, flash and resolder again. There are smd parts in the area of the connection pins so a desolder hot air gun was handy. 
![image_2022-10-25_18-46-09](https://user-images.githubusercontent.com/37833326/197857044-6a3a6273-8150-4402-b627-d07e2bdbb8ba.jpg)

So now i have tasmota 12.x successfully up and running, what info do you need next to get this thing flying?!
![image_2022-10-25_18-46-13](https://user-images.githubusercontent.com/37833326/197857084-47809241-a5f2-4419-b19a-4c83d778926c.jpg)
![image_2022-10-25_18-46-38](https://user-images.githubusercontent.com/37833326/197857097-42a178a8-9727-4d1e-8b48-a3dd6da7d70c.jpg)

![image_2022-10-25_18-52-35](https://user-images.githubusercontent.com/37833326/197858071-a8fb5020-d2c8-490b-a88b-2da3b33841f0.jpg)
![image_2022-10-25_18-52-45](https://user-images.githubusercontent.com/37833326/197858091-17da35b5-92f8-41fe-875a-021abf8c5d50.jpg)
![image_2022-10-25_18-52-41](https://user-images.githubusercontent.com/37833326/197858108-2e24f234-03a0-4f56-bae4-048b35603a53.jpg)

![image_2022-10-25_18-52-38](https://user-images.githubusercontent.com/37833326/197858126-87c8c5b9-13f5-4e4e-94c8-4a4ff7441020.jpg)

Sash

**Comment by AditaNi** [comment]
Happy new Year!

Which configuration do you use for Tasmota?

Regatds
Adrian

**Comment by jkwim** [comment]
can you post a picture of the underside of the pcb?

**Comment by jkwim** [comment]
Different PCB is shown here:

[https://www.youtube.com/watch?v=2QCFsCVEIAA](https://www.youtube.com/watch?v=2QCFsCVEIAA
)
![image](https://user-images.githubusercontent.com/84637540/210206834-98f7d4e6-1303-477e-b42d-29a37a64c060.png)

**Comment by arendst** [comment]
I don't understand why people install Tasmota on these devices.

Adding an ESP8266 or ESP32 with modbus interface enabled (#define USE_MODBUS_ENERGY) allows easy access to all (user configurable by rule file) modbus registers of these devices without losing the LCD functionality already built into the device.

**Reply by AditaNi** [reply]
Why do you need to have two wifi boards?
 Simply because you can?
And then maybe some people whants to have the data localy saved and not in some who knows whos clouds ...
I can continue but makes no sense....

**Reply by g6094199** [reply]
> I don't understand why people install Tasmota on these devices.
> 
> Adding an ESP8266 or ESP32 with modbus interface enabled (#define USE_MODBUS_ENERGY) allows easy access to all (user configurable by rule file) modbus registers of these devices without losing the LCD functionality already built into the device.

that is one point why more and more people install OpenBeken instead of Tasmota. 
the devs over here get lazy.

**Comment by makromat** [comment]
Do you know what is the voltage of pulse output? Thanks a lot
